index.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+    <html lang="en-US">
+	<head>
+	    <meta charset="UTF-8">
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+	    <title>Youtube Page</title>
+	    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	    <link rel="stylesheet" href="style/youtube-style.css">
+	    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet"> 
+	</head>
+	<body>
+	    <header>
+	        <nav>
+	            <div id="nav-leftpart">
+	                <i class="fa fa-navicon"  style="color: lightgray ; margin-left:6px"></i>
+	                <img class="logo" src="assets/youtube-logo.png" alt="youtube-logo">
+	            </div>
+	
+	            <form>
+	                <div id="search-area">
+	                <input type="search" id="search-bar" name="youtubesearch" placeholder="Search">
+	                <button id="submit" type="submit"><i class="fa fa-search"></i></button>
+	                </div>
+	            </form>
+	
+	            <div id="nav-rightpart">
+                    <img class="right-icons" id="upload-icon" src="assets/upload.png" alt="upload-icon">
+                    <img class="right-icons" src="assets/create-video.png" alt="create-video-icons">
+                    <img class="right-icons" src="assets/youtube-apps.png" alt="youtube-apps-icons">
+	            <button id="sign">Sign in</button>
+	            </div>
+	
+	        </nav>
+	    </header>
+
+	    <main id="container">
+        <section id="youtube-main">
+    	<iframe width="560" height="315" src="https://www.youtube.com/embed/bAyaXyYM3Eo" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+        <div id="video-info">
+            <h3 id="video-title"> Building On The Past </h3>
+    	
+    <div id="video-details">
+    		<img id="avatar" src="assets/avatar.jpg" alt="avatar">
```

Comment:
Your avatar image is not showing the issue is you have `src="assets/avatar.jpg"` instead of `src="assets/avatar.JPG"`

Reply:
Hi Ispirett,

Thanks for your review and your time. I have identified the issue about the avatar image. I have changed the file extension from .JPG to .jpg in the assets folder so that it matches the file extension showed on the code. 

Thanks